tic-tac-toe/ui/utils/callMethods.ts

```diff
@@ -84,69 +91,61 @@ export async function startNewGameMethod({ playerB }: NewProps) {
     sender,
     keyPair
   );
-
   const calimeroConnection = await nearAPI.connect({
     networkId: networkId,
     keyStore: keyStore,
     signer: signer,
-    nodeUrl: `${process.env.NEXT_PUBLIC_CALIMERO_NODE_URL}/90-calimero-testnet/neard-rpc/`,
-    walletUrl: "https://testnet.mynearwallet.com/",
+    nodeUrl: `${process.env.NEXT_PUBLIC_CALIMERO_NODE_URL}/${networkId}/neard-rpc/`,
+    walletUrl: `${process.env.NEXT_PUBLIC_WALLET_ENDPOINT_URL}`,
     headers: {
-      // @ts-expect-error: Type 'undefined' is not assignable to type 'string | number'
-      "x-api-key": process.env.NEXT_PUBLIC_CALIMERO_X_API_HEADER_KEY,
+      "x-api-key": process.env.NEXT_PUBLIC_CALIMERO_X_API_HEADER_KEY || "",
     },
   });
   let account = new nearAPI.Account(calimeroConnection.connection, sender);
   let contract = new nearAPI.Contract(account, contractAddress, {
     viewMethods: [],
     changeMethods: ["start_game"],
   });
-  await contract["start_game"](
-    {
-      player_a: sender.toString(),
-      player_b: playerB,
-    },
-    new big.BN("300000000000000"),
-    new big.BN("0")
-  );
+  try {
+    await contract["start_game"](
+      {
+        player_a: sender.toString(),
+        player_b: playerB,
+      },
+      new big.BN("300000000000000"),
```

Comment:
Can we add a comment about why this amount. Was this calculated using near-api-js. 
Would it make sense to create a game config and use it for CalimeroSDK and Near connections?

Reply:
Do we need it for this small example ? https://github.com/near/near-api-js/blob/master/packages/cookbook/utils/calculate-gas.js , this number is MAX_GAS value

Reply:
@frdomovic correct, 300 TGas is max possible gas, it is here so it passes always. In reality it can probably be 10-20 TGas, but I would leave it here as is. Maybe just extract the value to a const

Here is a sample of this tx how much it spent: https://explorer.testnet.near.org/transactions/8Ex1pojKw8fp5Y8X85gy5fTX9VyWrC4VUHCTvNR1kArT